subsys/mgmt/ec_host_cmd/Kconfig

```diff
@@ -9,20 +9,39 @@ config EC_HOST_CMD
 	bool "Support Embedded Controller host command handler subsystem"
 	help
 	  Enable host command processing for embedded controllers on notebook
-	  computers. Enabling this option requires specifying a chosen
-	  zephyr,ec-host-interface device as the ec host command backend that
-	  receive incoming host command requests to process.
+	  computers. Only version 3 is supported.
 
 if EC_HOST_CMD
 
+module = EC_HC
+module-str = ec-host-commands
+source "subsys/logging/Kconfig.template.log_config"
+
 config EC_HOST_CMD_HANDLER_STACK_SIZE
 	int "Stack size for the EC host command handler thread"
 	default 800
 
 config EC_HOST_CMD_HANDLER_TX_BUFFER
 	int "Buffer size in bytes for TX buffer shared by all EC host commands"
-	default EC_HOST_CMD_BACKEND_SHI_MAX_RESPONSE if EC_HOST_CMD_BACKEND_SHI
+	default 0 if EC_HOST_CMD_BACKEND_ESPI
+	default 0 if EC_HOST_CMD_BACKEND_SHI
 	default 256
+	help
+	  Buffer size in bytes for TX buffer defined by the host command handler.
+	  Some backend layers can define their own buffer, so the size can be zero not to
+	  duplicate buffers. If multiple backends are used, the size has to be set by user
```

Comment:
```suggestion
	  Some backend layers can define their own buffer, so the size can be zero to avoid
	  duplicate buffers. If multiple backends are used, the size has to be set by user
```